lib/public/views/QcFlags/details/qcFlagDetailsComponent.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { detailsList } from '../../../components/Detail/detailsList.js';
+import errorAlert from '../../../components/common/errorAlert.js';
+import { createButton } from '../../../components/common/form/createButton.js';
+import { deleteButton } from '../../../components/common/form/deleteButton.js';
+import { markdownDisplay, markdownInput } from '../../../components/common/markdown/markdown.js';
+import { LabelPanelHeaderComponent } from '../../../components/common/panel/LabelPanelHeaderComponent.js';
+import { PanelComponent } from '../../../components/common/panel/PanelComponent.js';
+import { tooltip } from '../../../components/common/popover/tooltip.js';
+import spinner from '../../../components/common/spinner.js';
+import { table } from '../../../components/common/table/table.js';
+import { BkpRoles } from '../../../domain/enums/BkpRoles.js';
+import scrollTo from '../../../utilities/scrollTo.js';
+import { qcFlagsVerificationsActiveColumns } from '../ActiveColumns/qcFlagVerificationsActiveColumns.js';
+import { qcFlagDetailsConfiguration } from './qcFlagDetailsConfiguration.js';
+import { h, iconWarning, sessionService } from '/js/src/index.js';
+
+/**
+ * Render a cancel verification button component
+ *
+ * @param {function} onclick function called when button is clicked (if null, button will be disabled)
+ * @return {Component} the button component
+ */
+export const cancelVerificationButton = (onclick) => h(
+    'button#cancel-verification.btn.btn-warning',
+    onclick ? { onclick } : { disabled: true },
+    'Cancel verification',
+);
+
+/**
+ * Render QC flag details for data pass page
+ *
+ * @param {QcFlagDetialsModel} qcFlagDetialsModel The overall model object.
+ * @param {object<string, funciton<*, string|component>>} additionalDetailsConfiguration display configuration for QC flag specific details
+ * @returns {Component} The details page
+ */
+export const qcFlagDetailsComponenet = (qcFlagDetialsModel, additionalDetailsConfiguration = {}) => {
+    const {
+        qcFlag: remoteQcFlag,
+        dplDetector: remoteDplDetector,
+        deleteResult,
+        verificationEnabled,
+        verificationCreationModel,
+    } = qcFlagDetialsModel;
+
+    return h('', [
+        h('.flex-row.justify-between.items-center', [
+            h('h2', 'QC Flag Details'),
+            h('.flex-row.g3', [
+                !verificationEnabled && h('.pv3', h(
```

Comment:
Given that you already have the information on the client side, I would recommend the `verify` button is disabled if the user that created is the one looking at it. 
Then if the user hovers it, a balloon appears that informs the user that he is the owner of the flag type and he cannot verify his own work.

This will save the user from doing the following actions if they do not pay attention:
- click verify
- add a verification comment
- click the 'Post Verify' button
- encounter the error message